### ISSUE TITLE
[worker reload] [worker init] Named export 'parseEntities' not found.

### ISSUE THREAD

**Issue by ArthurDanjou** [nuxt/content#1493]
### Environment

- Operating System: `Darwin`
- Node Version:     `v16.11.0`
- Nuxt Version:     `3.0.0-rc.9`
- Nitro Version:    `0.5.0`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `srcDir`, `router`, `modules`, `css`, `components`, `content`, `colorMode`
- Runtime Modules:  `@pinia/nuxt@0.4.1`, `nuxt-windicss@2.5.1`, `@nuxtjs/color-mode@3.1.4`, `@vueuse/nuxt@9.1.1`, `@nuxt/content@2.0.1`
- Build Modules:    `-`

### Reproduction

https://github.com/ArthurDanjou/artdanj-website-v2/tree/feature_content

### Describe the bug

When starting the dev server, I just receive the error caused by nuxt-content. In the main branch, I can run the server without any errors.

### Logs

```shell
yarn run v1.22.19
$ nuxt dev
Nuxi 3.0.0-rc.9                                                                                                                                                         17:27:56
Nuxt 3.0.0-rc.9 with Nitro 0.5.0                                                                                                                                        17:27:56
                                                                                                                                                                        17:27:59
  > Local:    http://localhost:3000/
  > Network:  http://192.168.1.29:3000/

ℹ Please create ~/components/content and restart the Nuxt server to use components in Markdown                                                            @nuxt/content 17:27:59

 WARN  autoImports:extend hook is deprecated. Use addImports() from @nuxt/kit or imports:extend with nuxt>=3.0.0-rc.9.                                                  17:27:59


 WARN  autoImports:sources hook is deprecated. Use imports:sources with nuxt>=3.0.0-rc.9.                                                                               17:27:59

ℹ nuxt-windicss v2.5.1 running with config: ./windi.config.ts.                                                                                            nuxt-windicss 17:27:59
ℹ Vite client warmed up in 735ms                                                                                                                                        17:28:00
ℹ Vite server warmed up in 69ms                                                                                                                                         17:28:01
✔ Nitro built in 523 ms                                                                                                                                           nitro 17:28:01

 ERROR  [worker reload] [worker init] Named export 'parseEntities' not found. The requested module 'file:///Users/arthurdanjou/Documents/Workspace/ares-v2/node_modules/parse-entities/index.js' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'file:///Users/arthurdanjou/Documents/Workspace/ares-v2/node_modules/parse-entities/index.js';
const { parseEntities } = pkg;


  import { parseEntities } from 'node_modules/parse-entities/index.js';
  ^^^^^^^^^^^^^
  SyntaxError: Named export 'parseEntities' not found. The requested module 'node_modules/parse-entities/index.js' is a CommonJS module, which may not support all module.exports as named exports.
  CommonJS modules can always be imported via the default export, for example using:

  import pkg from 'node_modules/parse-entities/index.js';
  const { parseEntities } = pkg;

  at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
  at async ModuleJob.run (node:internal/modules/esm/module_job:179:5)
  at async Loader.import (node:internal/modules/esm/loader:178:24)
  at async Object.loadESM (node:internal/process/esm_loader:68:5)
  at async handleMainPromise (node:internal/modules/run_main:63:12)
```

**Comment by farnabaz** [comment]
I think your repository is private and I can't access your repository. 
However, this issue should be fixed in [edge version](https://content.nuxtjs.org/guide/migration/edge-channel/). Feel free to test the edge release

**Comment by ArthurDanjou** [comment]
I've just changed the visibility of my repository. I will test the edge version later in the day ! Thank you for the information. I will come back to close the issue if my problem is fixed

**Comment by ArthurDanjou** [comment]
The edge channel is well working. But I think I don't understand how to fetch my page. For example, I want to write my uses page in markdown and import the page with nuxt content. The <ContentDoc path="uses" /> return an error in the console : 

```
[nuxt] [request error] [unhandled] [500] Cannot read properties of undefined (reading 'children')
  at visit (./node_modules/unist-util-visit-parents/index.js:116:20)
  at visitParents (./node_modules/unist-util-visit-parents/index.js:65:30)
  at visit (./node_modules/unist-util-visit/index.js:44:7)
  at ./.nuxt/dev/index.mjs:342:5
  ```
  
  You can find the content page here : https://github.com/ArthurDanjou/artdanj-website-v2/blob/feature_content/src/content/uses.md 
  and the vue page which fetch the page : https://github.com/ArthurDanjou/artdanj-website-v2/blob/feature_content/src/pages/uses.vue

**Comment by ArthurDanjou** [comment]
Fixed in 2.1.0
